pyproject.toml

```
@@ -0,0 +1,6 @@
+# Ruff configurations
+[tool.ruff]
+extend-exclude = ["data", "models", "db", "cache"]
+
+[tool.ruff.lint]
+select = ["E4", "E7", "E9", "F", "I", "T10", "CPY"]
```

Please add the `known-first-party` config for `isort`, which was already configured in the `.isort.cfg` file:
```suggestion
select = ["E4", "E7", "E9", "F", "I", "T10", "CPY"]

[tool.ruff.lint.isort]
known-first-party = ["bugbug_http"]
```

This should fix the unexpected changes in `http_service/tests/*`.

---

Done